docs/installation.adoc

```diff
@@ -154,3 +154,12 @@ __  ____  __  _____   ___  __ ____  ______
 2021-03-05 12:26:43,200 INFO  com.eno.kub.ope.Operator - Adding listener 'OperatorXpClientCacheInvalidate'
 2021-03-05 12:26:43,200 INFO  com.eno.kub.ope.Operator - Starting informers
 ----
+
+=== Delete xp from your cluster
```

Comment:
This title is confusing. Since you mention uninstalling the operator in this section. Should it be called `Uninstalling` or something?

Also, do not mention cluster role or cluster role bindings. These things should be cleaned up by the operator. So lets file an issue for this: https://github.com/enonic/xp-operator/issues/219